.devcontainer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# devcontainer-focused Rocker
+FROM ghcr.io/rocker-org/devcontainer/geospatial:4.2
```

Comment:
using Rocker's 'devcontainer' images are a bit more batteries-included for Codespaces.  Not sure we want codespaces yet but curious to explore.